src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -308,6 +308,7 @@ public static class Release
                 // Keep alphabetical
                 //
                 public static readonly string AgentReleaseDirectory = "agent.releaseDirectory";
+                public static readonly string AgentReleaseRootDirectory = "agent.releaseRootDirectory";
```

Comment:
there should be a location for tasks to write temp files and upload them as part of task execution independent of whether tasks runs as part of build or release. we should not let tasks to write contents under release root directory.